tests/failpoints/cases/test_early_apply.rs

```diff
@@ -1,92 +0,0 @@
-// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
-
-use raft::eraftpb::MessageType;
-use std::sync::atomic::{AtomicBool, Ordering};
-use std::sync::Arc;
-use std::thread;
-use std::time::*;
-use test_raftstore::*;
-
-/// Tests early apply is disabled for singleton.
-#[test]
-fn test_singleton_early_apply() {
```

Comment:
The case should be reserved to see if early apply is still working as expected.

Reply:
Ok, I will move it to `test_single` file

Reply:
I change them back and split it into two tests and add some comments

Reply:
I don't see them in diff. Did you commit it?

Reply:
My fault. I forget to use `git add`.